open-banking-accelerator/accelerators/ob-apim/repository/resources/apis/SelfCarePortal/scp-insequence.xml

```diff
@@ -0,0 +1,4 @@
+<sequence xmlns="http://ws.apache.org/ns/synapse" name="SCPTokenExchange">
+    <property name="Authorization" scope="transport" action="remove"/>
+    <property name="Authorization" value="Basic YWRtaW5Ad3NvMi5jb206d3NvMjEyMw==" scope="transport"/>
```

Comment:
Can we remove this auth value and use documentation to ask a developer to configure it

Reply:
fixed with https://github.com/wso2/financial-open-banking/pull/1/commits/80d467bc167898197adc085a069aa2f9091f16ba